lib/firespring_dev_commands/git.rb

```diff
@@ -218,7 +218,7 @@ def checkout(branch, dir: default_project_dir, raise_errors: false)
       # If the branch we are checking out doesn't exist, check out either the staging branch or the main branch
       actual_branch = branch
       unless branch_exists?(dir, branch)
-        actual_branch = [staging_branch, main_branch].uniq.find { |it| branch_exists?(dir, it) }
+        actual_branch = [default, staging_branch, main_branch].uniq.find { |it| branch_exists?(dir, it) }
         puts "Branch #{branch} not found, checking out #{actual_branch} instead".light_yellow
```

Comment:
Does the message need to include something if the default is also not found?

Reply:
I don't think so because of the checking out actual_branch part will tell you which branch was actually checked out. I can't think of good wording for if default isn't also found.